### DISCUSSION TITLE
Questions about deployment

### DISCUSSION THREAD

**Discussion by rose8928** [screego/server#185] [Q&A] [answered]
May I ask, has been deployed, there are no other problems, but the same network segment can be shared, different network segments can not be shared, why

**Comment by jmattheis** [comment]
What do you mean with `the same network can be shared` and `different network segments can not be shared`. Have you checked everything here: https://screego.net/#/faq?id=stream-doesnt-load

**Reply by rose8928** [reply]
Sorry, I mean, after my testing, it can be shared in the same wifi environment, if it is different, it cannot be shared. 
I have checked,but i can't understand the difference between stun and trun.today,i found some things.like this:
this is mine:
![image](https://github.com/screego/server/assets/35814595/c2d62f2e-2cfd-4ee2-ab67-810c0adb38dc)
this is app.screego.net
![image](https://github.com/screego/server/assets/35814595/7b56f91e-c10c-44dd-8bcf-954637bf6b69)
how to make 'stun' to 'turn'
this is my screego.config
[screego.config.txt](https://github.com/user-attachments/files/16092447/screego.config.txt)

**Reply by jmattheis** [reply, marked as answer]
Either configure users and login or set `SCREEGO_AUTH_MODE` to `none`.

**Reply by rose8928** [reply]
thank you!
